test/test_helper.rb

```diff
@@ -35,23 +35,9 @@
   end
 end
 
-class OpenStruct
-  def blank?
-    self == OpenStruct.new || super
-  end
-
-  def empty?
-    self == OpenStruct.new || super
-  end
-end
-
 class ActionDispatch::IntegrationTest
   include Rails.application.routes.url_helpers
 
-  def response
-    JSON.parse(@response.body, object_class: OpenStruct)&.data
-  end
```

Comment:
I had to remove this because it was attempting to parse every response as JSON. I poked at it a bit to switch on response type, but decided to remove it because we don't have anything json right now. There's some stuff here which is obviously inspired by patterns we don't have, eg `.data` at the end there.

If we later need a json-parsing helper here, I'm all for it.